tspng/extraction.py

```diff
@@ -25,7 +25,7 @@ def extract_from_file(path, mime_type=MIME_TYPE):
     dict=json.loads(meta[mime_type])
     return dict
 
-def extract_from_files(paths, mime_type=MIME_TYPE):
+def extract_from_files(paths: list, mime_type: str=MIME_TYPE):
```

Comment:
The `from typing import List` should be added to the top of the file and the `List` "type"/class should be used for the `paths` argument. Using the `list` type is actually a function not a type.

Reply:
Changes made as requested